content/en/platform/corda/5.0-dev-preview-2/getting-started/cordapp-standard-development-environment/csde.md

```diff
@@ -8,26 +8,40 @@ menu:
     weight: 2000
 section_menu: corda-5-dev-preview2
 ---
+{{< note >}}
+The CSDE is experimental. The decision whether or not we release it as part of Corda 5.0 will, in part, be based on your [feedback](https://community.r3.com/c/corda-5-developer-preview/41).  
+{{< /note >}}
+
 The CorDapp Standard Development Environment (CSDE) makes the process of prototyping CorDapps on Developer Preview 2 more straight-forward.
-The CSDE is obtained by cloning our `CSDE-Cordapp-Template-Kotlin` repository to your local machine. The CSDE provides:
-* A prepared CorDapp project that you can use as a starting point to develop your own prototypes
-* A base Gradle configuration that brings in the dependencies you need to write and test a Corda 5 CorDapp
-* A set of Gradle helper tasks which speed up and simplify the development and deployment process; these are effectively wrappers over the [Corda CLI](../../developing/corda-cli/overview.html)
-* Debug configuration for debugging a local Corda cluster
-* The `MyFirstFlow` code which forms the basis of the Getting Started documentation
-* The ability to configure the members of the local Corda network
+The CSDE is obtained by cloning our `CSDE-cordapp-template-kotlin` or `CSDE-cordapp-template-java` repository to your local machine. The CSDE provides:
+* a prepared CorDapp project that you can use as a starting point to develop your own prototypes.
+* a base Gradle configuration that brings in the dependencies you need to write and test a Corda 5 CorDapp.
+* a set of Gradle helper tasks which speed up and simplify the development and deployment process; these are effectively wrappers over the [Corda CLI](../../developing/corda-cli/overview.html).
+* debug configuration for debugging a local Corda cluster.
+* the `MyFirstFlow` code which forms the basis of the Getting Started documentation.
+* the ability to configure the members of the local Corda network.
 
 {{< note >}}
-The CSDE is experimental. The decision whether or not we release it as part of Corda 5.0 will, in part, be based on your [feedback](https://community.r3.com/c/corda-5-developer-preview/41).  
+The images in this section show the `CSDE-cordapp-template-kotlin` repository. The `CSDE-cordapp-template-java` repository looks similar but the flow code is defined in multiple source files. For more information, see [Java Flow Code](../first-flow/code-java.html).
```

Comment:
This this might be a better wording:
The images in this section are from opening the CSDE-cordapp-template-kotlin repository in IntelliJ. When opening the CSDE-cordapp-template-java repository in IntelliJ everything will look very similar.  The differences are that the Java code exists under a Java directory, the code spread over more source files and all references to the CSDE-cordapp-template-kotlin repository will be to the CSDE-cordapp-template-java repository. For more information, see Java Flow Code.

Reply:
Made another change :)